packages/block-editor/src/components/font-sizes/with-font-sizes.js

```diff
@@ -160,35 +155,40 @@ export default ( ...fontSizeNames ) => {
 						}
 
 						const newState = Object.entries(
```

Comment:
We're using the existing `Object.entries()` here and just moving the filtering after the conversion to an array.